llvm/test/Transforms/LoopIdiom/processloopstridedstore-drop-debugloc.ll

```diff
@@ -0,0 +1,92 @@
+; RUN: opt < %s -passes=loop-idiom -S | FileCheck %s
+
+; Function Attrs: noinline nounwind uwtable
+define dso_local void @fun(ptr noundef %a) #0 !dbg !10 {
+
+; CHECK-LABEL: entry:
+; CHECK-NOT: call void @llvm.memset.p0.i64{{.*}}dbg {{![0-9]+}}
+entry:
+  tail call void @llvm.dbg.value(metadata ptr %a, metadata !16, metadata !DIExpression()), !dbg !17
```

Comment:
If you still have the source, can you regenerate this with `-gmlt` so we don't get variable location metadata?